simplified-books-borrowing/src/main/java/org/nypl/simplified/books/borrowing/internal/BorrowSubtaskDirectory.kt

```diff
@@ -14,9 +14,10 @@ class BorrowSubtaskDirectory : BorrowSubtaskDirectoryType {
       BorrowAxisNow,
       BorrowBearerToken,
       BorrowCopy,
+      // BorrowSAMLDownload must precede BorrowDirectDownload in precedence.
+      BorrowSAMLDownload,
```

Comment:
This is a good point. The semantics with regards to priority in subtask selection is undefined right now. Having it just be in the order they were passed in is not great... Something we need to fix.